### PR TITLE
Explorer changes ABX-5668

### PR DESCRIPTION
**Pull request by mesabg**
This PR includes the following tickets

https://bullioncapital.atlassian.net/browse/ABX-5668
https://bullioncapital.atlassian.net/browse/ABX-7338
https://bullioncapital.atlassian.net/browse/ABX-7486

Please @MilesNorton feel free to review, and let me know

### CODE REVIEW

File: src/components/layout/Account.tsx [hunk 1]
```diff
@@ -26,12 +26,65 @@ class AccountPage extends React.Component<Props, State> {
   loadAccount = async () => {
     const accountId = this.props.match.params.id
 
-    // In the scenario required, should add /transactions to URI to access deactivated account
+    // Validate invalid account
     try {
-      const account = await getAccount(this.props.selectedConnection, accountId)
-      this.setState({ account })
+      const status: boolean = await validateAccount(accountId)
```
Comment (BorislavShekerov) [inline src/components/layout/Account.tsx:31]: This has become a bit hard to read, here is what I would suggest:

```
  loadAccount = async () => {
    const accountId = this.props.match.params.id

    try {
      const isAccountAddressValid: boolean = await validateAccount(accountId)
      if (!isAccountAddressValid) {
        return this.setState({ invalidAccount: true })
      } else {
        // In the scenario required, should add /transactions to URI to access deactivated account
        await this.getAccountDetailsOrUseEmptyBalanceAccount(accountId)
      }
    } catch (e) {
      this.setState({ invalidAccount: true })
      return
    }
  }

  getAccountDetailsOrUseEmptyBalanceAccount = async (accountId: string) => {
    try {
      const account = await getAccount(this.props.selectedConnection, accountId)
      this.setState({ account })
    } catch (e) {
      // A 404 response code will be returned when the address has been merged (during a deposit) and can't be found,
      // We want to display a balance of 0 in this case
      this.setState({
        account: createEmptyBalanceAccountRecord(accountId),
      })
    }
  }
```

You can create a new `empty_balance_account_creator.ts` in `utils` :

```
import { AccountRecord, TradeRecord } from 'js-kinesis-sdk'

export const createEmptyBalanceAccountRecord = (accountId: string) =>
  ({
    id: accountId,
    paging_token: '',
    account_id: accountId,
    sequence: 0,
    subentry_count: 0,
    thresholds: {
      low_threshold: 0,
      med_threshold: 0,
      high_threshold: 0,
    },
    flags: {
      auth_required: false,
      auth_revocable: false,
    },
    balances: [
      {
        balance: '0.0',
        asset_type: 'native',
      },
    ],
    _links: {},
    signers: [],
    data: {},
    effects: () =>
      Promise.resolve({ records: [], next: () => Promise.resolve(), prev: () => Promise.resolve() } as any),
    offers: () => Promise.resolve({ records: [], next: () => Promise.resolve(), prev: () => Promise.resolve() } as any),
    operations: () =>
      Promise.resolve({ records: [], next: () => Promise.resolve(), prev: () => Promise.resolve() } as any),
    payments: () =>
      Promise.resolve({ records: [], next: () => Promise.resolve(), prev: () => Promise.resolve() } as any),
    trades: () => Promise.resolve({ records: [], next: () => Promise.resolve(), prev: () => Promise.resolve() } as any),
  } as AccountRecord)

```

File: src/components/widgets/AccountInfo.tsx [hunk 2]
```diff
@@ -31,6 +31,18 @@ export class AccountInfo extends React.Component<Props, State> {
   }
 
   loadOperations = async (cursor?: string, limit: number = 10) => {
+    if (!this.props.account) {
```
Comment (BorislavShekerov) [inline src/components/widgets/AccountInfo.tsx:34]: import { isEmpty } from 'lodash'

```
if (!this.props.account || isEmpty(this.props.account.operations)) {
return this.setState({
        showLoadMore: false,
      })
}
```

File: src/components/layout/Account.tsx (hunk 1 above)
Comment (mesabg) [reply src/components/layout/Account.tsx]: Changes performed

File: src/components/widgets/AccountInfo.tsx (hunk 2 above)
Comment (mesabg) [reply src/components/widgets/AccountInfo.tsx]: Also here, all fixed

**Pull request merged**